src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -67,29 +70,33 @@
 
     private static final String UPDATED_SSCS = "Updated SSCS";
     public static final String DM_STORE_USER_ID = "sscs";
-
+    private static final Enum<EventType> EVENT_TYPE = EventType.UPLOAD_DOCUMENT;
```

Comment:
Is this the right Event Type here?

Reply:
The A/C call for that: Case [CCD_ID] with event uploadDocument added [documents] through MYA